src/test/groovy/com/github/benmanes/gradle/versions/DifferentGradleVersionsSpec.groovy

```diff
@@ -115,7 +115,8 @@ final class DifferentGradleVersionsSpec extends Specification {
       '5.4.1',
       '5.5.1',
       '5.6',
-      '6.0'
+      '6.0',
+      '6.1.1',
```

Comment:
Shouldn’t you have 6.2-rc-1 here?

Reply:
At first I did but realized I need a totally different test for the dependency verification. But yeah it doesn't hurt to check it here too.

Reply:
Oh I see, I missed it being explicit in the new test

Reply:
Just in case: `dependency verification` is a silent feature in Gradle 6.1.
In other words, it is there, however, it is not documented.

In other words, there might be users who have `gradle/verification-metadata.xml` and Gradle 6.1 at the same time.